src/status_im/ui/screens/chat/components/input.cljs

```diff
@@ -462,19 +462,18 @@
                                        :input-focus         #(input-focus text-input-ref)
                                        :set-active          set-active-panel}])])]]]))))
 
-(defn calculate-y [context keyboard-shown min-y max-y reply]
-  (let [added-value (if reply 38 0)]
-    (if keyboard-shown
-      (if (= (:state @context) :max)
-        (+ max-y added-value)
-        (if (< (:y @context) max-y)
-          (+ (:y @context) added-value)
-          (do
-            (swap! context assoc :state :max)
-            (+ max-y added-value))))
-      (do
-        (swap! context assoc :state :min)
-        min-y))))
+(defn calculate-y [context keyboard-shown min-y max-y added-value]
+  (if keyboard-shown
```

Comment:
added-value shouldn't be used here, instead, it should be used in `get-input-content-change`

Reply:
yeah, right, so then we could keep it in the `calculate-y` but apply only to `y` not to max